License.txt

```diff
@@ -1,5 +1,7 @@
 Copyright (c) 2015 Vijay
 
+Copyright 2014 rengwuxian https://github.com/rengwuxian/MaterialEditText
```

Comment:
We should check which is the proper mechanism to include the attribution to external code. 
This PR includes code from other project?